fastlane/Fastfile

```diff
@@ -0,0 +1,102 @@
+actions_path "actions/common/"
+actions_path "actions/common/cocoapods/"
+actions_path "actions/common/github/"
+actions_path "actions/common/sourcery/"
+actions_path "actions/common/json/"
+actions_path "actions/common/utils/"
+
+
+desc "Generate VideoRenderer release description"
+lane :release_videorenderer_description do 
+	description = "Template description"
+	begin
+		description = github_changelog_generator(output: "unreleased.md", unreleased_only: true)
+		UI.message description
+		description
+	rescue => exception
+		UI.user_error! exception.message
+	end
+end
+
+desc "Check if VideoRenderer version exist"
+lane :is_release_exist do |options|
+	draft_release = github_releases_api(
+		repo_name: options[:repo_name],
+		action: "get_github_releases", 
+		tag: options[:version])
+	UI.message("#{draft_release}")
+	draft_release != nil	
+end
+
+# Usage like these expected 'fastlane release_videorenderer next_version:1.18 swift_version:3.2'
+desc "Release VideoRenderer version"
+lane :release_videorenderer do |options|
+	next if !utils(action: "is_branch_up_to_date")
+	version = options[:next_version]
+	UI.user_error!("Expecting `next_version`!") unless version != nil
+
+	repo_name = "aol-public/OneMobileSDK-videorenderer-ios"
+
+	UI.user_error!("Release on tag #{version} already exists!") unless !is_release_exist(repo_name: repo_name, version: version)
+
+	swift_version = options[:swift_version]
+	UI.user_error! "Expecting `swift_version`!" unless swift_version != nil
+
+	travis_branch = options[:branch]
```

Comment:
What is the purpose of this check - this will be only triggered from the developers machine.